src/loaders/OMETiffLoader.js

```diff
@@ -36,8 +36,11 @@ export default class OMETiffLoader {
     this.height = this.omexml.SizeY;
     this.tileSize = firstImage.getTileWidth();
     const { SubIFDs } = firstImage.fileDirectory;
```

Comment:
```suggestion
    const { subIFDs } = firstImage.fileDirectory;
```
Why capital?

Reply:
This is a tiff standard.

Reply:
(i.e I don't have control over it.  Between the OMEXML and the tiff, there are a fair number of examples of this sort of capitalization wonkiness).

Reply:
Sorry not to read that more carefully: Should we save it to a lower cased variable?
```suggestion
    const subIFDs = firstImage.fileDirectory['SubIFDs'];
```